packages/lesswrong/components/hooks/useForeignCrosspost.ts

```diff
@@ -59,17 +60,32 @@ export const useForeignCrosspost = <Post extends PostWithForeignId, FragmentType
   }
 
   const apolloClient = useForeignApolloClient();
+
+  // const getCrosspostQuery = gql`
+  //   query GetCrosspostQuery($args: JSON) {
+  //     getCrosspost(args: $args)
+  //   }
+  // `;
+
+  // const args = {
+  //   ...fetchProps,
+  //   documentId: localPost.fmCrosspost.foreignPostId
+  // };
+
+  // const { data, loading, error } = useQuery(getCrosspostQuery, { variables: { args } });
```

Comment:
Yeah, there's unfortunately an extra hop (in the case where the page is being rendered on the client rather than SSR), but we don't currently have a way to make a non-GraphQL request from components in a way that behaves correctly for both client-side rendering and SSR.  Shouldn't be that much of a perf hit, since as you said, the extra request is the one between servers (not between client & server) and the servers should be pretty close to each other.